pkg/network/tracer/tracer_windows.go

```diff
@@ -239,8 +240,8 @@ func (t *Tracer) DebugNetworkMaps() (*network.Connections, error) {
 // DebugEBPFMaps is not implemented on this OS for Tracer
 //
 //nolint:revive // TODO(WKIT) Fix revive linter
-func (t *Tracer) DebugEBPFMaps(maps ...string) (string, error) {
-	return "", ebpf.ErrNotImplemented
+func (t *Tracer) DebugEBPFMaps(w io.Writer, maps ...string) error {
```

Comment:
The following fixes the revive linter warning, so you can remove the nolint line above
```suggestion
func (t *Tracer) DebugEBPFMaps(_ io.Writer, _ ...string) error {
```